scripts/smoke_test_examples.py

```
@@ -0,0 +1,384 @@
+                for file_path in subdir_path.rglob("*.py"):
+                    if file_path.name not in SKIP_FILES and "diamond" not in file_path.name.lower():
+                        example_files.append(file_path)
+
+        # Also check for standalone examples in the main examples directory
+        for file_path in self.examples_dir.glob("*.py"):
+            if file_path.name not in SKIP_FILES:
+                example_files.append(file_path)
+
+        return sorted(example_files)
+
+    def run_example_with_timeout(self, example_path: Path) -> Tuple[bool, str, str]:
```

Ruff (0.11.9)

Undefined name `Tuple`

(F821)